thirdparty/runtime_environments.go

```diff
@@ -355,10 +405,20 @@ func (c *RuntimeEnvironmentsClient) getHistory(ctx context.Context, opts history
 	defer resp.Body.Close()
 	if resp.StatusCode != http.StatusOK {
 		msg, _ := io.ReadAll(resp.Body)
-		return nil, errors.Errorf("HTTP request returned unexpected status '%s': %s", resp.Status, string(msg))
+		apiErr := errors.Errorf("HTTP request returned unexpected status '%s': %s", resp.Status, string(msg))
+		grip.Error(message.WrapError(apiErr, message.Fields{
+			"message":     "bad response code from image visibility API",
+			"params":      params,
+			"status_code": resp.StatusCode,
+		}))
+		return nil, apiErr
 	}
 	amiHistory := &APIHistoryResponse{}
 	if err := gimlet.GetJSON(resp.Body, &amiHistory); err != nil {
+		grip.Error(message.WrapError(err, message.Fields{
+			"message": "parsing response from image visibility API",
```

Comment:
I think if you want to just make one alert for runtime environments, you should keep the message consistent for every message (or add something else consistent, like `"reason": "runtime environment API alerting"` -- that also tells us why we're even doing this; otherwise, i would recommend adding a super tiny ADR or comments to the function headers, since we may forget longterm why we even have this alerting).